src/main/jjtree/net/sf/jsqlparser/parser/JSqlParserCC.jjt

```diff
@@ -1616,9 +1616,13 @@ Expression RegularCondition():
 	| "<" { result = new MinorThan(); }
 	| "=" { result = new EqualsTo(); }
 	| ">=" { result = new GreaterThanEquals(); }
+	| "> =" { result = new GreaterThanEquals("> ="); }
 	| "<=" { result = new MinorThanEquals(); }
+	| "< =" { result = new MinorThanEquals("< ="); }
 	| "<>" { result = new NotEqualsTo(); }
+	| "< >" { result = new NotEqualsTo("< >"); }
```

Comment:
I'll provide an updated commit to address:
a. The 'any amount of whitespace' issue (between parts of the operators)
b. Removal of "space between ! and =", as this means something different in PostgreSQL